smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSink.java

```diff
@@ -140,6 +142,15 @@ public KafkaSink(KafkaConnectorOutgoingConfiguration config, KafkaCDIEvents kafk
 
         log.connectedToKafka(client.getClientId(), config.getBootstrapServers(), topic);
 
+        KafkaAttributesExtractor kafkaAttributesExtractor = new KafkaAttributesExtractor();
+        InstrumenterBuilder<KafkaTrace, Void> builder = Instrumenter.builder(
+                GlobalOpenTelemetry.get(),
+                "io.smallrye.reactive.messaging.opentelemetry",
```

Comment:
Should the instrumentation name here be `io.smallrye.reactive.messaging.kafka` as used by `KafkaConnector#TRACER`?

Reply:
The instrumentation name indicates the instrumentation library name. It depends if we want to be more fine-grained or not.